docs/druid-0.9.1-lookup-migration.md

```diff
@@ -4,18 +4,18 @@ IAP 1.3.0 that ships with Druid 0.9.1 introduces a new way of making query time
 The [new](http://druid.io/docs/0.9.1/development/extensions-core/lookups-cached-global.html) and [old](http://druid.io/docs/0.9.1/development/extensions-core/namespaced-lookup.html) lookups are incompatible.
```

Comment:
Czy ten plik jest potrzebny ? Bo to staroć jakiś jest :)

Reply:
Ustaliłem wczoraj z Marcinem, że sprzątanie dokumentacji zrobimy kiedy indziej.